src/push/kibana_api.ts

```diff
@@ -37,7 +37,12 @@ import {
 import { generateURL } from './utils';
 
 // Default chunk size for bulk put / delete
-export const CHUNK_SIZE = 100;
+export const CHUNK_SIZE = parseInt(process.env.CHUNK_SIZE) || 100;
```

Comment:
Can we move this somewhere before the pipeline, lets say during the push command invocation rather than here - https://github.com/elastic/synthetics/blob/41588ed325a9a351a891f72eb8ecaeaa4580d52c/src/push/index.ts#L68 or https://github.com/elastic/synthetics/blob/41588ed325a9a351a891f72eb8ecaeaa4580d52c/src/cli.ts#L223